src/get-started/java-binding.md

```diff
@@ -362,6 +392,24 @@ service:
 - Follow the instructions in the [application guide][app-tutorial] to configure
   and start an Exonum node with your service.
 
+## Common Library
+
+Java Binding includes a library module that can be useful for Java clients and
+does not have a dependency on Java Binding Core. The module contains Java
+classes obligatory for core that can now as well be easily applied in clients,
+if necessary. In this case the classes are simply launched by a user as a library.
```

Comment:
> In this case the classes are simply launched by a user as a library.

I'd leave this out because users do not need an explanation of what a library is.